chore(linter): remove unused commented-out code

shulaoda opened the pull request:
It's been around ten months and this code hasn't been used during that time.
To avoid redundancy, we can remove it.

Sysix commented:
### Merge activity

* **May 25, 3:32 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

shulaoda committed:
chore(linter): remove unused commented-out code (#11283)

It's been around ten months and this code hasn't been used during that time.
To avoid redundancy, we can remove it.
